Lib/test/_test_multiprocessing.py

```diff
@@ -653,6 +653,47 @@ def test_forkserver_sigkill(self):
             self.check_forkserver_death(signal.SIGKILL)
 
 
+class TestStdOutAndErr(unittest.TestCase):
```

Comment:
Perhaps you can move these tests closer to the test added by the other PR:
https://github.com/python/cpython/pull/4073/files#diff-b046ab474480855fb4a01de88cfc82bbR585

I also wonder if those tests can be unified. They seem to be doing very similar things.

Reply:
My 1st guess was to update that test case too. But someone has disabled that test case by adding a _ prefix to the class name. Also the test module itself is prefixed by _. So these tests are not run automatically. And I couldn't make that test case work.
Also that test seemed to test the inter process Event handling, so I guessed I would stick to the strict and often unpractical "only test one thing at a time" principle.
I think the "_test_multiprocess.py" modul should be refurbished but I don't think that I am the right person for that.

Reply:
Ha. You misunderstood how `_test_multiprocessing` works :-) The test classes there are used as base classes for the actual tests in `test_multiprocessing_fork`, `test_multiprocessing_forkserver` and `test_multiprocessing_spawn`. So you just have to run one of those three test modules to get the tests to work!

Reply:
Whoa! :) Now it all makes sense! I will make yet another commit (3rd) and unify the test cases.